src/arrays/chunk.js

```diff
@@ -0,0 +1,9 @@
+function chunk(array, size) {
+  const results = [];
+  while (array.length) {
+    results.push(array.splice(0, size));
+  }
```

Comment:
Well,
Main mistake in current PR (which is existing in other methods too) is usage of the `Array.prototype` methods.
As it is said in the task:
![image](https://user-images.githubusercontent.com/20482331/130402509-f5746dc3-fdce-4c38-987f-9e4671ef8690.png)

And as you can see here [Array.prototype.slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) is prototype method.

So This have to be fixed.